index.html

```diff
@@ -9256,7 +9264,20 @@ <h5>Presentational Role Inheritance</h5>
 			<rdef>textbox</rdef>
 			<div class="role-description">
 				<p>A type of input that allows free-form text as its value.</p>
+				<p>Authors MUST limit its children to completely presentational elements (like a spacer image, decorative graphic, or clearing element).</p>
 				<p>If the <pref>aria-multiline</pref> <a>attribute</a> is <code>true</code>, the <a>widget</a> accepts line breaks within the input, as in an <abbr title="Hypertext Markup Language">HTML</abbr> <code>textarea</code>. Otherwise, this is a simple text box. The intended use is for languages that do not have a text input <a>element</a>, or cases in which an element with different <a>semantics</a> is repurposed as a text field.</p>
+				
+				<p>
+					User agents MUST expose the value of elements with role <code>textbox</code> to <a>assistive technologies</a>.
+					The value of a <code>textbox</code> is represented by one of the following:
+					</p>
+					<ul>
+						<li>If the <code>textbox</code> element is a host language element that provides a value, such as an HTML <code>input</code> element, the value of the textbox is the value of that element.</li>
+							<li>Otherwise, the value of the <code>textbox</code> is represented by its descendant elements and can be determined using the same method used to compute the name of a <rref>button</rref> from its descendant content.</li>
+					</ul>
+
+
+
```

Comment:
And these empty lines, as they are unnecessary

Reply:
@spectranaut I couldn't find these blank lines and they are not showing up in the diff for my latest commit [81cd343](https://github.com/chlane/aria/commit/81cd343772f02835c1d27cbf45553726508ee064)

Reply:
@spectranaut found them and removed them thanks. See https://github.com/w3c/aria/pull/1827/commits/55fca79365055aa062e17600503afca9c5b48362